Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
+
+extension ExtendedForReactiveness {
+	/// A proxy which exposes the reactivity of `self`.
+	public var rac: Reactive<Self> {
```

Comment:
If we weight clarify above brevity - which is what the Swift API guideline prefers - then yes, `reactive` is a sensible choice.